include/systems/equation_systems.h

```diff
@@ -454,6 +454,24 @@ class EquationSystems : public ReferenceCountedObject<EquationSystems>,
    **/
   void allgather ();
 
+  /**
+   * Calls to reinit() will also do two-step coarsen-then-refine
+   **/
+  void enable_refine_in_reinit()
```

Comment:
Oh, I'm an idiot.  I was looking at the main page, not the whole diff, and didn't see the disable_ function.

Okay, IMHO this is fine as is.